### PR TITLE
chore(release): 4.0.2

### PR DESCRIPTION
**Pull request by TarikGul**
Release v4.0.2:

Updated:
`@polkadot/api => v4.2.1`
`@polkadot/apps-config => v0.85.1`

Added
`@substrate/dev` for config consolidation.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,18 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [4.0.2](https://github.com/paritytech/substrate-api-sidecar/compare/v4.0.1...v4.0.2) (2021-03-15)
+
+
+### Bug Fixes 
+
+* **types** Update @polkadot/api to get latest substrate types for `Extender{Header, SignedBlock}`, and updated metadata
+* **types** Update @polkadot/apps-config to get latest substrate types for `Extender{Header, SignedBlock}`, and updated metadata
```
Comment (emostov) [inline CHANGELOG.md:11]: ``
```suggestion
* **types** Update @polkadot/apps-config to get latest chain specific types
```

Comment (emostov) [reply CHANGELOG.md:11]: The primary purpose of apps-config is chain specific type definitions; api is for substrate and polkadot types

**Review by emostov — APPROVED** [review]
LGTM modulo one nit

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,18 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [4.0.2](https://github.com/paritytech/substrate-api-sidecar/compare/v4.0.1...v4.0.2) (2021-03-15)
```
Comment (emostov) [inline CHANGELOG.md:5]:
```suggestion
## [4.0.2](https://github.com/paritytech/substrate-api-sidecar/compare/v4.0.1...v4.0.2) (2021-03-22)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,18 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [4.0.2](https://github.com/paritytech/substrate-api-sidecar/compare/v4.0.1...v4.0.2) (2021-03-22)
+
+
+### Bug Fixes 
+
+* **types** Update @polkadot/api to get latest substrate types for `Extender{Header, SignedBlock}`, and updated metadata
+* **types** Update @polkadot/apps-config to get latest chain specific types
```
Comment (dvdplm) [inline CHANGELOG.md:11]: Do you think it's worth it to state the actual versions here?

**Pull request merged**